kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -641,6 +641,380 @@ paths:
+      summary: get connector status
+      operationId: getConnectorStatus
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: connectorName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ConnectorStatus'
+
+  /api/clusters/{clusterName}/connectors/{connectorName}/restart:
+    post:
+      tags:
+        - /api/clusters/connectors
+      summary: restart the connector
+      operationId: restartConnector
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: connectorName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: OK
+        409:
+          description: rebalance is in progress
+
+  /api/clusters/{clusterName}/connectors/{connectorName}/pause:
+    put:
+      tags:
+        - /api/clusters/connectors
+      summary: pause the connector
+      operationId: pauseConnector
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: connectorName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        202:
+          description: Accepted
+
+  /api/clusters/{clusterName}/connectors/{connectorName}/resume:
+    put:
+      tags:
+        - /api/clusters/connectors
+      summary: resume the connector
+      operationId: resumeConnector
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: connectorName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        202:
+          description: Accepted
+
+  /api/clusters/{clusterName}/connectors/{connectorName}/tasks:
+    get:
+      tags:
+        - /api/clusters/connectors
+      summary: get connector tasks
+      operationId: getConnectorTasks
+      parameters:
+        - name: clusterName
+          in: path
+          required: true
+          schema:
+            type: string
+        - name: connectorName
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ConnectorTask'
+
+  /api/clusters/{clusterName}/connectors/{connectorName}/tasks/{taskId}/status:
+    get:
```

Comment:
Could we include task status to task object?

Reply:
Merged into one, thanks